warp10/src/main/java/io/warp10/script/functions/ACCELREPORT.java

```diff
@@ -0,0 +1,41 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.functions;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptStackFunction;
+import io.warp10.standalone.StandaloneAcceleratedStoreClient;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+
+public class ACCELREPORT extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+  
```

Comment:
Is it a debug function or not ? If so, it should report more than the last fetch status. Maybe a map with :
- accelerator activation status
- accelerator preload status
- accelerator max time depth
- last fetch was accelerated
- number of gts in the accelerator (help for debug remote customers too) (as directory is not affected, FIND won't make any difference when called after ACCEL.NOPERSIST)